utils/fixed_decimal/src/decimal.rs

```diff
@@ -1245,7 +1308,8 @@ impl FixedDecimal {
     /// ```
     pub fn signum(&self) -> Signum {
```

Comment:
Note: The second half of the issue should be to reflect the new convention in `FixedDecimalFormat`. When you do that refactoring, you may be able to remove `Signum`.

I think that the `SignDisplay` algorithm in FixedDecimalFormat should either be folded into a new function here on `FixedDecimal` or removed from ICU4X.

Reply:
Agreed, I had started looking at what signum was for and I don’t think it makes much sense with the new structure.